homeassistant/components/tesla_fleet/sensor.py

```diff
@@ -378,6 +378,20 @@ class TeslaFleetTimeEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.POWER,
         entity_registry_enabled_default=False,
     ),
+    SensorEntityDescription(
+        key="island_status",
+        options=[
+            "island_status_unknown",
+            "on_grid",
+            "off_grid",
+            "off_grid_unintentional",
+            "off_grid_intentional",
+        ],
+        device_class=SensorDeviceClass.ENUM,
+    ),
+    SensorEntityDescription(
+        key="storm_mode_active",
```

Comment:
From the name it sounds like it should be a `binary_sensor` e.g. status is `True` or `False`?

Reply:
Yeah, it does certainly looks binary
https://github.com/home-assistant/core/blob/dev/tests/components/tesla_fleet/fixtures/live_status.json#L16

Reply:
@JEMcats how about you remove it from this PR, as I am going to drop it in Teslemetry and Tessie at the same time in https://github.com/home-assistant/core/pull/126704

Reply:
I have removed the Storm Watch active sensor. [6eb19f6](https://github.com/home-assistant/core/pull/126438/commits/6eb19f642ab0d519430f8aded5d829f24e5a86ba)